cmd/root/exercise_test.go

```diff
@@ -82,17 +88,17 @@ func Test_renderText(t *testing.T) {
 		}
 		got := testModel.renderText()
 		if got != test.want {
-			t.Fatalf("%s failed\nexpected\n%s\ngot\n%s",
-				test.name, test.want, got,
+			t.Fatalf("%s failed\nwant\n%s\n%q\ngot\n%s\n%q\n",
```

Comment:
Fatalf or Errorf?

- Fatalf stops test output if there's an error. No other tests are run
- Errorf continues running tests, despite failures

I can control which tests are failing and which are succeeding by passing the test function to the `go test` command, if there's too much stuff going on. However, if I fix things, and then break other things, it's difficult to catch that earlier if I use Fatalf.

I should change these tests in this file to Errorf instead.